Doc/library/dialog.rst

```diff
@@ -0,0 +1,230 @@
+Tkinter Dialogs
```

Comment:
A bit late to the party, but may I suggest that this file be renamed to `tkinter.dialog.rst`?